kurtosis-module/impl/services/wallet.ts

```diff
@@ -179,7 +178,10 @@ function generateJsSrcUpdatingCommands(jsVars: Map<string, string>): Result<stri
         // Parcel envvars get set as a bunch of properties, like:
         // ....MOONPAY_API_URL:"SOMETHING",ACCOUNT_ID_SUFFIX:"SOMETHING ELSE"....
         // We therefore look for the property assignments and overwrite the value with our constant string instead
-        const updateJsFileCommand = `sed -i -E 's${JS_REPLACEMENT_SED_DELIMITER}([,{])${key}:"[^"]*"([,}])${JS_REPLACEMENT_SED_DELIMITER}\\1${key}:"${value}"\\2${JS_REPLACEMENT_SED_DELIMITER}g' ${WALLET_JS_FILE_GLOB}`
+        const srcRegexp = `([,{])${key}:[^,]*([,}])`
+        const replacementRegexp = `\\1${key}:"${value}"\\2`
+        log.debug(`Replacing variable '${key}' to '${value}' using regexp: '${srcRegexp}'`);
+        const updateJsFileCommand = `sed -i -E 's${JS_REPLACEMENT_SED_DELIMITER}${srcRegexp}${JS_REPLACEMENT_SED_DELIMITER}${replacementRegexp}${JS_REPLACEMENT_SED_DELIMITER}g' ${WALLET_JS_FILE_GLOB}`
```

Comment:
Some values are now JS functions, so we need the regexp to be a bit wider. AFAICT this works for now

Reply:
Can we add this as a comment, explaining why the regex is broader?